client/package.json

```diff
@@ -48,26 +48,27 @@
     "firebase": "^4.12.0",
     "js-yaml": "^3.8.4",
     "ng2-slim-loading-bar": "^4.0.0",
-    "ngx-clipboard": "^9.0.0",
-    "ngx-monaco": "0.6.0",
-    "rxjs": "^5.5.6",
+    "ngx-clipboard": "11.0.0",
+    "ngx-monaco": "0.8.0",
+    "rxjs": "6.1.0",
+    "rxjs-compat": "6.1.0",
```

Comment:
Do we really need the compat library? Or is it for dependencies who rely on our `rxjs` version and still use the old syntax?

Reply:
Exactly. Shared code is still on an older version, so is server/admin-cli/cli etc.